src/pymatgen/io/vasp/inputs.py

```diff
@@ -435,6 +435,8 @@ def from_str(
         # This is in line with VASP's parsing order that the POTCAR
         # specified is the default used.
         if default_names is not None and symbols is not None:
+            vasp5or6_symbols = True
```

Comment:
I don't think this tag is doing anything as `symbols is not None` effectively filter VASP4 cases, but for some reason enumlib test might fail, so there must have been some cases that are missed